docs/getting_started/generate_token.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Generate Auth Token
+sidebar_position: 5
+---
+
+ Authentication token is needed for custom dapps and for CLI usage. Authentication tokens strengthen account security by verifying users identity. These unique codes offer protection from unauthorized access. This token will authenticate and authorize external applications to communicate with your shard. Calimero is a permissioned chain and it allows for granular control over the token's time to live and associated permissions.
```

Comment:
`Authentication tokens strengthen account security by verifying users identity.`
Which accounts are we talking about here?